assets/src/editor/blocks/landing-page-hero/index.js

```diff
@@ -86,12 +86,12 @@ export const settings = {
 		buttonText: {
 			type: 'string',
 			source: 'html',
-			selector: '.hero__call-to-action',
+			selector: '.hero__cta-button',
```

Comment:
The selector `.hero__cta-button` isn't in the markup output from the save() method... should this value be changing? It looks like it's still `.hero__call-to-action`.